fuzzydir.lua

```diff
@@ -124,10 +124,17 @@ end
 function fast_readdir(path)
     if powershell_version >= 3 then
         return call_command({
-            "powershell.exe",
+            "powershell",
             "-NoProfile",
             "-Command",
-            "& { Get-ChildItem -LiteralPath FileSystem::\"" .. path .. "\" -Directory | foreach { $_.Name } }",
+            [[
+            $dirs = Get-ChildItem -LiteralPath ]] .. string.format("%q", path) .. [[ -Directory
+            foreach ($n in $dirs.Name) {

```

Comment:
Not sure if this syntax works to be honest - at least doesn't seem to with PS 2 I have in a VM. Though, this optimization doesn't work in general with PS 2 as it lacks the `-Directory` flag for `Get-ChildItem`, so maybe it's fine with newer versions?..

In any case, could you please change it to something like this just to be safe?
```
foreach($dir in $dirs) {
    $u8clip = [System.Text.Encoding]::UTF8.GetBytes($dir.Name)
    ...
}
```

We also don't really need the trailing slash, it's purely aesthetical.

Reply:
Done